source/Engine.cpp

```diff
@@ -2322,15 +2331,31 @@ void Engine::AddSprites(const Ship &ship)
 {
 	bool hasFighters = ship.PositionFighters();
 	double cloak = ship.Cloaking();
+	bool damageHighlight = Preferences::Has("Damage highlights");
 	bool drawCloaked = (cloak && ship.IsYours());
+	bool drawHeat = ship.Heat() > 0.9;
+	bool drawShield = ship.RecentShield() > 4.;
+	double shield = sqrt(ship.RecentShield()/ship.Attributes().Get("shields"));
+	double heat = min((ship.Heat()-0.9) / 16, 1.);
```

Comment:
```suggestion
	double heat = min((ship.Heat()  - .9) / 16, 1.);
```
Just a bit of spacing
Btw 0.9 could be written .9 since its the same.
Also I think it should be put in the nameless namespace at the start of engine as a constant (HIGH_HEAT or smth I'm not good with names)